conf/broker.conf

```diff
@@ -1014,6 +1014,8 @@ bookkeeperClientMinAvailableBookiesInIsolationGroups=
 # Enable/disable having read operations for a ledger to be sticky to a single bookie.
 # If this flag is enabled, the client will use one single bookie (by preference) to read
 # all entries for a ledger.
+# Please notice that sticky reads aren’t used unless ensemble size equals write quorum size for the ledger's
+# persistence policies.
```

Comment:
Nit: the double negative might be confusing. Here is my attempt at an alternative wording, but mine doesn't feel completely right either.

```suggestion
# Please notice that enabling sticky reads also requires that the ensemble size equals write quorum size for the ledger's
# persistence policies.
```